zeppelin-web/src/components/noteName-create/notename.controller.js

```diff
@@ -35,22 +36,23 @@ angular.module('zeppelinWebApp').controller('NotenameCtrl', function($scope, $ro
   };
 
   vm.preVisible = function(clone) {
-    var generatedName = vm.generateName();
-    $scope.note.notename = 'Note ' + generatedName;
+    $scope.note.notename = vm.generateName();
```

Comment:
This should be used as vm instead:
`vm.newNoteName` (will need to be also changes in the html: `notenamectrl.newNoteName`)